src/main/webapp/iaResults.jsp

```diff
@@ -1444,22 +1445,22 @@ function displayAnnotDetails(taskId, res, num, illustrationUrl, acmIdPassed) {
                         var rec=viewer.world.getItemAt(0).imageToViewportRectangle(ft.parameters.x*marginFactor*scale, ft.parameters.y*marginFactor*scale, width/marginFactor*scale, height/marginFactor*scale);
                 	   	viewer.viewport.fitBounds(rec);
                 	});
-                	
 
-                	
+
+
                 	//add this viewer to the global Map
                 	viewers.set(taskId+"+"+acmId,viewer);
                 	features.set(acmId, ft);
-            	
-            	
+
+
             	$('#task-' + taskId + ' .annot-' + acmId).addClass("seadragon");
-                
-                
+
+
             } else {
                 $('#task-' + taskId + ' .annot-' + acmId).append('<img src="images/no_images.jpg" style="padding: 5px" />');
             }
-            if (mainAsset.dateTime) {
-                imgInfo += ' <b>' + mainAsset.dateTime.substring(0,16) + '</b> ';
+            if(res.responseJSON.annotations[0] && res.responseJSON.annotations[0].encounterDate){
```

Comment:
just wanted to note (as this was my suggestion, in particular, using the `0`th element of `.annotations`), that there is a slight chance the 0th annotation will not necessarily correspond to `mainAsset`; in which case the date will be shown from an incorrect encounter.  however, i am not really sure how/if this case will ever come up.. but must have in some weird cases historically because of [this code](https://github.com/WildMeOrg/Wildbook/blob/master/src/main/webapp/iaResults.jsp#L1351-L1355) which basically sets `mainAsset` as _the first asset_ referenced by `annotations` array.   almost always this is going to be in the 0th annotation, tho.  so i think this fix is good enough.  

```
if (!res.responseJSON.annotations[i].asset) continue;  //no asset, meh continue
  ...
[if mainAsset not defined] mainAsset = res.responseJSON.annotations[i].asset
```